src/components/Contact.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable */
```

Comment:
- [ ] I do not think this file is related to this particular project, kindly remove it to make your project more professional.